src/index.js

```diff
@@ -16,16 +18,45 @@ const getItemsfromLocal = JSON.parse(localStorage.getItem('localStorageTasks'));
 const listContainer = document.querySelector('.todo-list');
 
 if (getItemsfromLocal != null) {
+  listContainer.textContent = '';
   getItemsfromLocal.forEach((anItem) => {
-    listContainer.innerHTML += `
-    <div class="item-container" id="${anItem.index}">
-      <div class="item-name" onclick="editData(${anItem.index})">${anItem.description}</div>
-      <button type="button" class="remove-btn" onclick="removeItem(${anItem.index})">Remove</button>
-    </div>
-    `;
+    // const funcParam = anItem.index
+
+    const itemContainer = document.createElement('div');
+    itemContainer.classList.add('item-container');
+    itemContainer.setAttribute('id', anItem.index);
+
+    const itemCheck = document.createElement('input');
+    itemCheck.classList.add('item-check');
+    itemCheck.setAttribute('type', 'checkbox');
+    itemCheck.checked = anItem.completed;
+    // eslint-disable-next-line no-undef
+    itemCheck.onclick = () => checkingItems(anItem.index);
+
+    const itemName = document.createElement('div');
+    itemName.classList.add('item-name');
+    // eslint-disable-next-line no-undef
+    itemName.onclick = () => editData(anItem.index);
+    itemName.textContent = anItem.description;
+
+    const remvButton = document.createElement('button');
+    remvButton.classList.add('remove-btn');
+    remvButton.setAttribute('type', 'button');
+    remvButton.onclick = () => removeItem(anItem.index);
+
+    const remImg = document.createElement('img');
+    remvButton.appendChild(remImg);
+    remImg.src = binIcon;
+
+    itemContainer.append(itemCheck, itemName, remvButton);
+    listContainer.appendChild(itemContainer);
+
```

Comment:
- [ ] **Good job so far 👏 👏** However, you have disabled the linters in your codes eg: ```// eslint-disable-next-line no-undef```, Kindly remove these comments and make sure that your eslinter is passing 🙏 

_By disabling your linter, you are effectively turning off a tool that is designed to catch errors and improve code quality. This can increase the risk of introducing errors into your codebase, which can lead to bugs and other issues down the line 😂._

Reply:
@TuyishimireEric 

If I do not disable it, it throws an error stating that editData and checkingItems are not defined. Could you please suggest a way to remove this error in any other way?

Reply:
And I have only disabled it for these 2 particular lines. It is not for the whole document.

Reply:
hi @TuyishimireEric 

I am waiting for your response

Reply:
Hey @sameedmohsin28, 

Sorry for the late reply 🙏 
I understand you are having a linter error by removing that comment.
However, if the error is related to undefined variables editData and checkingItems, you can try declaring them before using them in your code. 😎

or you may also try to refactor your codes by only repositioning those functions and I think this will help

and I think it would be better also to resubmit for re-review without waiting for my response 😎